### ISSUE TITLE
Support  for `--listEmittedFiles` + new message `file_emitted`

### ISSUE THREAD

**Issue by pp0rtal** [gilamran/tsc-watch#138]
### Summary: watch tsc generated files

I'm using the client and I would like to be notified everytime a new file is written.
This is possible with `listEmittedFiles` options which list the files emitted https://www.typescriptlang.org/tsconfig#listEmittedFiles

**Why?** Performance I'm using this lib client inside some compilation flow (there are some post-process after the Ts is compiled)
- Without: I need to watch the dist folder, which is resource consuming
- With `listEmittedFiles` I can unwatch the dist folder :)

### RFC: listEmittedFiles support

- I suggest to **activate** this ability by an iso `tsc-watch --listEmittedFiles` new option
- tsc output will be parsed obviously
- we trigger a new message `file_emitted` with the absolute file path

FYI tsc output is trivial:
```
TSFILE: /home/portal/.../config/config.js
TSFILE: /home/portal/.../config/config.js.map
TSFILE: /home/portal/.../tsconfig.tsbuildinfo
```

Example of implementation:
```typescript
tsWatch.start(...);
tsWatch.on("file_emitted", (emittedFile) => {
    console.log({ emittedFile });
});
```

### Impact
**no regression at all**
_(`--listEmittedFiles` has to be explicitely called)_

:information_source: Note I will do the PR

**Comment by pp0rtal** [comment]
Thank you @gilamran for such reactivity! :heart_eyes: https://www.npmjs.com/package/tsc-watch/v/4.6.1

**Comment by gilamran** [comment]
published on version 4.6.2
